Build v1 01

mohamedg-croesus opened the pull request:
Version 1.01 de RCTGAN

mohamedg-croesus committed:
modified:   README.md
modified:   rctgan/ctganpc/synthesizers/ctganpc.py
deleted:    rctgan/deepecho/__init__.py
deleted:    rctgan/deepecho/data/demo.csv
deleted:    rctgan/deepecho/demo.py
deleted:    rctgan/deepecho/models/__init__.py
deleted:    rctgan/deepecho/models/base.py
deleted:    rctgan/deepecho/models/basic_gan.py
deleted:    rctgan/deepecho/models/par.py
deleted:    rctgan/deepecho/sequences.py
new file:   rctgan/images/logo_rctgan.PNG
modified:   rctgan/relational/rctgan.py
deleted:    rctgan/timeseries/__init__.py
deleted:    rctgan/timeseries/base.py
deleted:    rctgan/timeseries/deepecho.py
modified:   setup.py
deleted:    setup.py.bak
deleted:    tasks.py.bak

mohamedg-croesus committed:
modified:   README.md

mohamedg-croesus committed:
modified:   README.md
new file:   rctgan/images/croesus-lab-horizontal-rouge.png

mohamedg-croesus committed:
modified:   README.md
modified:   rctgan/ctganpc/synthesizers/ctganpc.py

mohamedg-croesus committed:
modified:   HISTORY.md
modified:   rctgan/rctgan.py
modified:   setup.py

mohamedg-croesus committed:
modified:   README.md

sylvestrer-croesus reviewed (changes requested):
Quelques questions à propos du contenu des commentaires.

Je vais potentiellement modifier le review après avoir exécuté les étapes du readme.

mohamedg-croesus committed:
modified:   README.md
modified:   rctgan/relational/rctgan.py
modified:   setup.py

mohamedg-croesus committed:
modified:   README.md

mohamedg-croesus committed:
modified:   README.md
modified:   rctgan/ctganpc/synthesizers/ctganpc.py
modified:   rctgan/relational/rctgan.py
modified:   rctgan/tabular/ctganpc.py
